client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceConfigOverlay.less

```diff
@@ -0,0 +1,28 @@
+:local(.StartInstanceDetailsOverlay) {
+
+  .overlay__body {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .intro {
+    margin: 0 0 4px 0;
+    font-size: 13px;
+    color: var(--color-grey-225-10-55);
+  }
+
+  fieldset {
```

Comment:
Is this a generic (each form in any dialog works the same) concern? If not, why?